.github/workflows/docker-latest.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image for latest release
+
+on:
+  push:
+    tags:
+      - release
+
+env:
+  TEST_TAG: zeek:latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
```

Comment:
Similar here ... is this a preference or do we need it? (Fine with either, just trying to understand.)

Reply:
I cleaned this up so we check out submodules with `action/checkout`, and removed `docker/setup-qemu-action` since we do not intend to build e.g., arm images ATM. I also added a comment to `docker/setup-buildx-action` which is actually required.